app/uk/gov/hmrc/crdlcacheadminfrontend/auth/AuthController.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.crdlcacheadminfrontend.auth
+
+import javax.inject.Inject
+import play.api.mvc.MessagesControllerComponents
+import scala.concurrent.Future.successful
+import uk.gov.hmrc.crdlcacheadminfrontend.controllers.routes
+import uk.gov.hmrc.crdlcacheadminfrontend.auth.routes as authRoutes
+import uk.gov.hmrc.internalauth.client.FrontendAuthComponents
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+class AuthController @Inject()(auth: FrontendAuthComponents, authActions: AuthActions, mcc: MessagesControllerComponents)
+  extends FrontendController(mcc) {
+    given FrontendAuthComponents = auth
+
+    def signIn() = Action.async { implicit request =>
```

Comment:
Hmm I think that by using `Action.async` as well as the `auth.authorizedAction` we are nesting `Action`s - it's not a huge problem but I'm not sure that we need to do it.